### ISSUE TITLE
ERROR: BAVC1011932_part1.mov is expected but missing. 

### ISSUE THREAD

**Issue by iamdamosuzuki** [mipops/dvrescue#430]
Hi, I got a fatal error running a file through DVPackager. This file was run on a DSR-1500. The deck was having issues when this tape was created. Another deck was used later and that file worked fine, so something is inherently broken here. Though I thought it'd be good to know what's causing a total failure to package. 

DVRescue being used is the very latest version: 20220118-3/

File is currently uploading here (will be done ready in about 2 hours): https://drive.google.com/drive/folders/1Jp8FTqlxghbyhyvZX-bYlSZgE9jdB9j5?usp=sharing 

Here's the output of the verbose mode

```
Last login: Tue Jan 18 17:56:35 on ttys000
preservation@macmini-qc ~ % dvpackager -e mov -v /Volumes/SymplyUltra/TransferProjects/PV21_Arhoolie/01_Ingest/DV/review/BAVC1011932/BAVC1011932.dv
#CONDENSE_OPTS is set to -s
#Using /usr/local/opt/ffmpegdecklink/bin/ffmpeg-dl for ffmpeg operations.
Packaging BAVC1011932.dv, making dvrescue outputs
/usr/local/bin/dvpackager: line 1156: 51815 Segmentation fault: 11  "${DVRESCUE_PATH}" "${DVFILE}" "${OUTPUTOPTS[@]}"
/var/folders/13/93045l0d799_rvjxvc4bwflh0000gp/T/dvpackager.XXXXXX.ZANT7XC3:1.1: Document is empty

^
#No xmlstarlet error
#DV_FRAME_COUNT=()
#SOURCE_FORMAT=()
#Trying to present a table of ranges
failed to load external entity "/Volumes/SymplyUltra/TransferProjects/PV21_Arhoolie/01_Ingest/DV/review/BAVC1011932/BAVC1011932.dv.dvrescue.xml"

# Segmentation options. Split on: Recording_Start_Marker = 0, Recording_Timestamp_Jump = 0, Timecode_Jump = 0, Audio_characteristics_change = 0, Aspect_Ratio_change = 1
#         St='Flagged Start of a recording', ncTC='non-continuous timecode value', ncR='non-continuous recording timestamp value'
  # | PTS Range                         | Duration | Frame Range         | Byte Range                | Timecode    | Recording Timestamp    | Size      | Frame Rate | DAR   | ChSub | Audio     | St | ncTC | ncR |
  1 |                 -                 |    0.000 |        0 -        0 |           0 -           0 |             |                        |           |            |       |       | 2ch     0 |    |      |     |

The results will be written to /Volumes/SymplyUltra/TransferProjects/PV21_Arhoolie/01_Ingest/DV/review/BAVC1011932
failed to load external entity "/Volumes/SymplyUltra/TransferProjects/PV21_Arhoolie/01_Ingest/DV/review/BAVC1011932/BAVC1011932.dv.dvrescue.xml"
### Packaging started: /Volumes/SymplyUltra/TransferProjects/PV21_Arhoolie/01_Ingest/DV/review/BAVC1011932/BAVC1011932_part1.mov
(standard_in) 1: parse error
(standard_in) 1: parse error
failed to load external entity "/Volumes/SymplyUltra/TransferProjects/PV21_Arhoolie/01_Ingest/DV/review/BAVC1011932/BAVC1011932.dv.dvrescue.xml"
#The audio range sublist (from frame  to ) is
#
(standard_in) 1: parse error
(standard_in) 1: parse error
ERROR: ffmpeg and dvrescue disagree about the audio characterstics at offset  at .
We shall use ffmpeg's analysis on this. (ff:0 != dvr:-)
Consider updating to the latest version of dvrescue and this error recurs,
then consider mentioning this at https://github.com/mipops/dvrescue/issues.
FYI: Padding a little silence near  for BAVC1011932_part1.mov since a source portion ends with frames missing audio (via dvrescue).
#Writing a chapter track at /var/folders/13/93045l0d799_rvjxvc4bwflh0000gp/T/dvpackager.XXXXXX.IHItYCH2.ffmetadata.
failed to load external entity "/Volumes/SymplyUltra/TransferProjects/PV21_Arhoolie/01_Ingest/DV/review/BAVC1011932/BAVC1011932.dv.dvrescue.xml"
#Found 1 point(s) for chapter marks.
#Skipping the process to embed chapters since there would only be one.
#Trying to run: /usr/local/opt/ffmpegdecklink/bin/ffmpeg-dl  -y -nostdin -hide_banner -ss  -i /Volumes/SymplyUltra/TransferProjects/PV21_Arhoolie/01_Ingest/DV/review/BAVC1011932/BAVC1011932.dv -map 0:v:0 -c:v:0 copy -f rawvideo - | ffmpeg  -y -nostdin -hide_banner -i -   -filter_complex aevalsrc=channel_layout=stereo:sample_rate=48000:exprs=0|0:duration=[1at];[1at]concat=n=1:v=0:a=1[aud] -t  -c:a pcm_s16le -c:v:0 copy -f mov -map 0:v:0 -map [aud]  /Volumes/SymplyUltra/TransferProjects/PV21_Arhoolie/01_Ingest/DV/review/BAVC1011932/BAVC1011932_part1.mov
Invalid duration specification for ss: 
pipe:: Invalid data found when processing input
ERROR: BAVC1011932_part1.mov is expected but missing. Please retry with the -v option and report to https://github.com/mipops/dvrescue/issues.
ERROR: BAVC1011932.dv contained  DV frames, but the outputs contain 0 frames of DV and 0 were skipped
preservation@macmini-qc ~ % 

```

**Comment by libbyhopfauf** [comment]
hey @iamdamosuzuki ! Just out of curiosity because I was having issues with dvpackager for this version as well (both in the GUI and the CLI) have you downloaded 20220126 to see if it was resolved? I'm also curious because I've noticed some files that I have transferred using my DSR-1500 seem to give packager a lot of problems, but the only pattern I can find is where the timecode is missing (ish?) but that's not very consistent. Love that little deck, but it definitely does some weird things.
